WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -689,6 +719,34 @@ public void onPostChanged(OnPostChanged event) {
     public void onPostUploaded(OnPostUploaded event) {
         if (isAdded() && event.post.getLocalSiteId() == mSite.getId()) {
             loadPosts(LoadMode.FORCED);
+
+            final PostModel post = event.post;
+            if (event.isError()) {
+                showSnackbar(getString(R.string.editor_draft_saved_locally));
+            } else {
+                if (post != null) {
+                    boolean isDraft = post != null && PostStatus.fromPost(post) == PostStatus.DRAFT;
```

Comment:
strictly addressed in 657d4c6
...but then realized we were already accessing `event.post` at the very top of the `OnPostUploaded` method, so I decided to simplify it in 026974b